### PR TITLE
feat(lambda): Add lambda actions

### PR DESCRIPTION
**Pull request by andrewbackes**
This update adds Lambda permission checks and scanning of Kotlin files. The bulk of the lines added are sample files for tests. The actual code changes are very small.

### CODE REVIEW

**Review by edgarulg — APPROVED** [review]
Changes looks good to me but something I think would be good to have in the future is a test suite that asserts the policy created by the python scripts has the correct roles/permissions/etc.

The changes to include lambda and add it to the policy looks correct to me.

**Review by link108 — APPROVED** [review]
changes seem reasonable ✅

**Comment by andrewbackes** [comment]
@edgarulg While it isn't every combination of permissions, this test is meant to cover what you mentioned. It scans the sample files and verifies that the correct policy was created for the actions in those files.

https://github.com/armory/spinnaker-aws-policy/pull/20/files#diff-cd67fedca3d984a76d0eb8e654f108c78137d4c8e2d07adb6f4ca992aebda293R11

File: src/tests/test_generate.py [hunk 1]
```diff
@@ -21,6 +21,7 @@ def test_all(self):
             "ec2:DescribeAvailabilityZones",
             "ec2:DescribeSecurityGroups",
             "ec2:DescribeVpcs",
+            "lambda:DeleteFunction",
             "iam:PassRole"
         }
         self.assertEquals(expected, set(json_policy['Statement'][0]['Action']))
```
Comment (andrewbackes) [inline src/tests/test_generate.py:27]: @edgarulg

**Pull request merged**